docker-compose.yaml

```diff
@@ -16,7 +16,7 @@ services:
     environment:
       - DATABASE_HOST
       - MYSQL_ROOT_PASSWORD
-      - MYSQL_DATABASE
+      - MYSQL_DATABASE=cherrytwist
```

Comment:
Presumably the simple auth config also needs to be passed in here?

Reply:
It is passed, the context of the env variable is the container. Given the parameters are passed from .env file and the name of the env variable is the same, I am configuring it directly in docker-compose.